build/Dockerfile.debian.dev

```diff
@@ -0,0 +1,61 @@
+# Purpose of this container image is to ship pocket binary with additional
+# tools such as dlv, curl, etc.
+
+ARG TARGET_GOLANG_VERSION=1.18
+
+FROM golang:${TARGET_GOLANG_VERSION}-bullseye AS builder
+
+ENV PROTOC_VERSION 3.19.4
+ENV GOLANG_PROTOBUF_VERSION 1.28
+ENV MOCKGEN_VERSION 1.6.0
+ENV PATH $PATH:$GOPATH/bin
+
+### Install dependencies
+# Debian packages
+RUN apt-get update -qq && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential zip wget ca-certificates curl net-tools && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt
+
+# protoc
```

Comment:
For my personal learning: where do you come up (or reference) this approach to protoc installation?

Reply:
This part has been influenced by official golang docker images as they are (I knew for sure) are built for many cpu architectures.